test/new-e2e/utils/e2e/client/agent.go

```diff
@@ -90,13 +95,19 @@ func (a *Agent) WaitForReadyTimeout(timeout time.Duration) error {
 	interval := 100 * time.Millisecond
 	maxRetries := timeout.Milliseconds() / interval.Milliseconds()
 	err := backoff.Retry(func() error {
-		isReady, err := a.IsReady()
+		statusOutput, err := a.vmClient.ExecuteWithError(a.GetCommand("status"))
+		if err != nil {
+			return err
+		}
+
+		isReady, err := newStatus(statusOutput).isReady()
 		if err != nil {
 			return err
 		}
 		if !isReady {
 			return errors.New("agent not ready")
 		}
+
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Thank you !